install_config/storage_examples/ceph_rbd_dynamic_example.adoc

```diff
@@ -161,132 +116,122 @@ metadata:
      storageclass.beta.kubernetes.io/is-default-class: "true"
 provisioner: kubernetes.io/rbd
 parameters:
-  monitors: 192.168.1.11:6789,192.168.1.12:6789,192.168.1.13:6789  <1>
-  adminId: admin  <2>
-  adminSecretName: ceph-secret  <3>
-  adminSecretNamespace: kube-system  <4>
+  monitors: 192.168.1.11:6789,192.168.1.12:6789,192.168.1.13:6789 <1>  
+  adminId: admin <2>
+  adminSecretName: ceph-secret <3>
+  adminSecretNamespace: kube-system <4>  
   pool: kube  <5>
   userId: kube  <6>
-  userSecretName: ceph-user-secret  <7>
+  userSecretName: ceph-user-secret <7>
+----
+<1> Ceph monitors, comma delimited. This value is required.
+<2> Ceph client ID that is capable of creating images in the pool. The default is `admin`.
+<3> Secret Name for `adminId`. This value is required. The provided secret must have type `kubernetes.io/rbd`.
+<4> The namespace for `adminSecret`. The default is `default`.
+<5> Ceph RBD pool. The default is `rbd`, but this value is not recommended.
+<6> Ceph client ID used to map the Ceph RBD image. The default is the same as `adminId`.
+<7> The name of the Ceph Secret for `userId` to map the Ceph RBD image. It must exist in the
+same namespace as the PVCs. This is required unless it is set as the default in new projects.
```

Comment:
"This is required unless it is set as the default in new projects." I'd replace "it" with exactly what you're referring to and make this sentence active. "Unless you set the Ceph secret as the default in new projects, you must provide this parameter value."

Should "Secret" be capitalized?

I might make it clear that you explain how to do this later. Or add an xref.